onnx/defs/logical/old.cc

```diff
@@ -16,6 +16,7 @@ inline void logicalOpInference_opset1(InferenceContext& ctx) {
 std::function<void(OpSchema&)> BinaryLogicDocGenerator_opset1(
     const char* name) {
   return [=](OpSchema& schema) {
+#ifndef __ONNX_NO_DOC_STRINGS
```

Comment:
This kind of setup of the doc doesn't help in removing these strings from the binary. Hence the check for __ONNX_NO_DOC_STRINGS here.